data/en/2021/hispano/agenda.yml

```diff
@@ -0,0 +1,65 @@
+complete: true
```

Comment:
You'll want to set this to false, as this is what is printing the empty linked VOD buttons. We added this flag so that people who have prerecorded sessions can launch them whenever they like. Other than that looking good!